simple/stats/stat_var_hierarchy_constants.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Google Inc.
```

Comment:
can this be schema_constants.py?

Reply:
also should this be under `stats` vs. `stat_var` directory?

Reply:
Renamed file.

There are dependencies on things in `stats` (like `Triple`) so keeping it here.